core/grpc/proto/kms.v1.proto

```diff
@@ -697,3 +697,60 @@ message HealthStatusResponse {
   NodeType node_type = 6;
 
 }
+
+message InitiateResharingRequest {
+  // The request id of this resharing request (use to derive the session IDs and get the status)
+  RequestId request_id = 1;
+  // The active KMS context (default to DEFAULT_MPC_CONTEXT if empty)
+  RequestId context_id = 2;
```

Comment:
should we add a epoch_id here too? to be consistent with the other endpoints

Reply:
yeah the other ones all ignore it because we wanted the the API to be as stable as possible from the get go, i suppose